app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -350,14 +355,16 @@ def parse_load_balancer_health_check_member(member)
   end
 
   def parse_floating_ip(ip)
-    address = uid = ip.public_ip
+    cloud_network_only = ip.domain["vpc"] ? true : false
```

Comment:
ok, heads up, I tested it and yes, it returns a string. This would work because 
```ruby
"vpc"['vpc'] == 'vpc'
"standard"['vpc'] == nil
```

but thats not really intention revealing :)

so do you mind changing it to

```ruby
cloud_network_only = ip.domain == "vpc" ? true : false
```